src/_includes/social-proof.njk

```diff
@@ -1,7 +1,6 @@
 <ul>
     <li class="w-[46px] sm:w-[62px]">{% image "./images/home-logos/arduino-logo.png", "Arduino logo", [62] %}</li>
     <li class="w-[135px] sm:w-[156px]">{% image "./images/home-logos/leclanche-logo.png", "Leclanché logo", [156] %}</li>
-    <li class="w-[70px] sm:w-[93px]">{% image "./images/home-logos/lilly-logo.svg", "Lilly logo", [113] %}</li>
```

Comment:
@Yndira-E Please remove the image from the repo too

Reply:
Done!